app/src/ui/changes/changes-list.tsx

```diff
@@ -377,6 +379,30 @@ export class ChangesList extends React.Component<
     showContextualMenu(items)
   }
 
+  private getNamesOfSelectedOversizedFiles = () => {
+    const fileNames: string[] = []
+    const workingDirectoryFiles = this.props.workingDirectory.files
+
+    for (const file of workingDirectoryFiles) {
+      const fileIsSelected =
+        file.selection.getSelectionType() !== DiffSelectionType.None
+
+      if (fileIsSelected) {
+        try {
+          const filePath = path.join(this.props.repository.path, file.path)
+          // eslint-disable-next-line no-sync
+          const fileStatus = FSE.statSync(filePath)
+          const fileSizeMegabytes = fileStatus.size / 1000000
+          if (fileSizeMegabytes > 100) {
+            fileNames.push(file.path)
+          }
+        } catch (error) {}
```

Comment:
Do we want to get any sort of tracing into the logs around these errors.

At the least I'd recommend a `log.debug` so we can identify things in the dev tools:

```ts
} catch (error) {
  log.debug(`Unable to get file size for file ${filePath}`, error)
}
```

Reply:
Fantastic, I will make these changes!
Thanks again for the great feedback!